contents/blog/what-to-ask-in-interviews.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2022-06-08
+title: Really important interview questions that more engineers should ask
```

Comment:
Can we have a title that makes it clear that this is about job interviews, not just general interviews generally?

e.g. 'Really important questions that engineers should ask when interviewing for a job' (but maybe more slick)

Also I know that our target is engineers, but this particular content is clearly applicable to any type of role - for example, I'd expect questions like 'how do you handle on call' (we get asked this a lot) or questions around how they approach product development specifically. 

A more relevant title is probably something like 'Really important questions you should ask when interviewing at a startup'.

I'd change the title to not be engineering-specific personally.

Reply:
(Especially as the very last question is clearly a bit jokey)

Reply:
I agree with this, but think the suggestions are still a bit unclear. 

"Really important questions more engineers should ask when they are _being interviewed_" 

Feels like the most clear inversion of the current title. Some more clickbaity but shorter options:

"Questions nobody asks when they're interviewed'
"X Questions engineers should really ask during an interview"

Reply:
Have put a suggestion in. I think it's good.